src/main/java/duke/Task.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+/**
+ * A general task that a user can input.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Public constructor for a task.
+     *
+     * @param description Description of the event.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
```

Comment:
I think this JavaDoc should begin with a verb? e.g. "Instantiates" or "Constructs". I believe this applies for your other constructors as well.

Reply:
Thanks for pointing this out, did not know this haha. Will do the changes accordingly :)